web/resources/js/keyboard-shortcuts.js

```diff
@@ -0,0 +1,18 @@
+import hotkeys from 'hotkeys-js';
+
+hotkeys('option+c', function (event) {
+    window.location = "/user/create";
+});
+hotkeys('shift+option+c', function (event) {
+    window.open("/user/create");
+});
+hotkeys('option+f', function (event) {
+    window.location = "/user/search";
+});
+hotkeys('shift+option+f', function (event) {
+    window.open("/user/search");
+});
+hotkeys('option+h', function (event) {
+    window.open("https://www.youtube.com/embed/Ita6WyQ3SFQ?autoplay=1&controls=0");
+});
+
```

Comment:
I know this is personal taste, but I would prefer using a no-shiftkey-scheme like github uses: `option` (or `alt`) can clash with e.g. window managers, while just typing `p s` („pycroft search“) or whatever will always work (except on input fields, but that's restricted even with prefixes appearently).

Reply:
I accept your critique regarding window manager shortcuts, but the ones I know mostly use the mod key (Window/Tux symbol, depending on the keyboard). Additionally, simple key commands will likely conflict with browser plugins that allow sane surfing (e.g. Vimium and Tridactyl).

The best option is of course to finish the API so that an ncurses client may be developed :D

Reply:
@Lodifice I believe KDE has some things bound to `Meta-*`, and apart from that it is the default option for triggering menu hotkeys. If I press `M-h` on this very page on an uncustomized firefox-developer installation, it opens the “help” menu. Besides that, I know more websites that embrace a single-key setup than using a prefix – I make heavy use of this in e.g. tweetdeck, github and gitlab. If you use vimperator, wouldn't you find a scheme like this more intuitive? Wouldn't it be possible to avoid clashes by using a prefix like `p`, `x` or similar?

> The best option is of course to finish the API so that an ncurses client may be developed :D
Indeed.

Reply:
Don't confuse `M-h` and `<Alt> h` (the latter does indeed open the “help” menu on my Firefox 63); `hotkeys-js` does not seem to support proper Meta keys. Furthermore, don't break keybindings in my browser: Github, among others, rebinds `/`, and Tridactyl contains special-case code to unbreak searching for text in pages.